### ISSUE TITLE
Tap elsewhere doesn't close dropdown

### ISSUE THREAD

**Issue by samselikoff** [RSSchermer/ember-rl-dropdown#8]
Thanks for this great addon!

Clicking outside of an expanded dropdown closes it, but tapping (chrome, iOS) doesn't. Have you encountered this before, or is it a bug?

**Comment by RSSchermer** [comment]
That's definitely a bug, I'll look into it!

**Comment by RSSchermer** [comment]
I've created a new branch that should fix this: close-on-tap. Could you change the ember-rl-dropdown dependency in your package.json to use this branch:

```
"ember-rl-dropdown": "https://github.com/RSSchermer/ember-rl-dropdown#close-on-tap"
```

And tell me if that indeed fixes it for you and is usable?

**Comment by samselikoff** [comment]
Getting an error when running `npm` install

```
npm ERR! fetch failed https://github.com/RSSchermer/ember-rl-dropdown.git#close-on-tap
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 406
```

**Comment by RSSchermer** [comment]
Hmm, could you try "github notation" instead?

```
"ember-rl-dropdown": "rsschermer/ember-rl-dropdown#close-on-tap"
```

**Comment by samselikoff** [comment]
Awesome, works great!

**Comment by RSSchermer** [comment]
The `close-on-tap` branch is merged into master as of 0.4.x.
